### PR TITLE
109

### PR DESCRIPTION
**Pull request by Aylr**

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -1,51 +1,77 @@
...
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
 
 requirements:
   build:
     - python
-    - numpy >=1.11.2+mkl
```
Comment (levithatcher) [inline conda-recipe/meta.yaml]: Thanks for all this great work! Do we need a version for setuptools?

File: conda-recipe/meta.yaml [hunk 2]
```diff
@@ -1,51 +1,77 @@
...
-  # version: "1.0"
-  # the version is actually being set by the __conda_version__.txt file
-  # http://conda.pydata.org/docs/build.html#post-build-version
+  version: "0.1.8"
+
+source:
+  fn: healthcareai-0.1.8.tar.gz
+  url: https://pypi.python.org/packages/3f/f7/6c008469634ced5fe72730c6603cecfb1d6c2633428e27ae9b6344e307fb/healthcareai-0.1.8.tar.gz
+  md5: 9b7fdf2cbbf117e2eaf14043a4bf533b
+#  patches:
```
Comment (levithatcher) [inline conda-recipe/meta.yaml]: Ooh love that md5--starting to feel official

File: conda-recipe/meta.yaml (hunk 1 above)
Comment (Aylr) [inline conda-recipe/meta.yaml]: Not that I'm aware of...

**Comment by Aylr** [comment]
This does not include the official anaconda distribution. That will be researched at a later time.

**Pull request merged**